### ISSUE TITLE
Bad behaviour with symmetric intervals

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalRootFinding.jl#34]
```
julia> f(xx) = ( (x,y) = xx; sin(x)*sin(y) )
f (generic function with 1 method)

julia> rts = roots(∇(f), IntervalBox(-5..5, 2), Newton)
1-element Array{IntervalRootFinding.Root{IntervalArithmetic.IntervalBox{2,Float64}},1}:
 Root([0, 0] × [0, 0], :unique)
```

**Comment by dpsanders** [comment]
This probably needs some clever way to choose the midpoint when bisecting, e.g. just choose it slightly off from the centre.

**Comment by dpsanders** [comment]
Fixed by #55 by bisecting off-centre.
